code/datums/components/spybug.dm

```diff
@@ -0,0 +1,42 @@
+/datum/component/spybug
+	var/obj/item/clothing/glasses/sunglasses/spy/linked_glasses
```

Comment:
Components have a much much better variable for the linked item called parent.

Reply:
I did not want to make two separate objects (one ninja star, the other pen) and have them share the similar code.

Putting them under the same class is impossible since they both have different behaviors.

What workaround do you suggest?